langchain/vectorstores/deeplake.py

```diff
@@ -245,10 +247,18 @@ def ingest(sample_in: list, sample_out: list) -> None:
             elements[i : i + batch_size] for i in range(0, len(elements), batch_size)
         ]
 
+        num_workers = min(self.num_workers, len(batched) // max(self.num_workers, 1))
+
+        checkpoint_interval = self.checkpoint_interval
+        if checkpoint_interval is None:
```

Comment:
could there be cases where users explicitly don't want to checkpoint (and so the None should be respected)?